### PR TITLE
Gitpod: Add Better-TOML

### PR DESCRIPTION
**Pull request by JesterOrNot**
Add syntax highlighting for TOML
Before
Screenshot 2020-02-18 at 9 47 20 AM
After
image

### CODE REVIEW

File: .gitpod.yml [hunk 1]
```diff
@@ -7,17 +7,17 @@ github:
...
+# enable for pull requests coming from forks (defaults to false)
     pullRequestsFromForks: true
-    # add a "Review in Gitpod" button as a comment to pull requests (defaults to true)
+# add a "Review in Gitpod" button as a comment to pull requests (defaults to true)
     addComment: true
-    # add a "Review in Gitpod" button to pull requests (defaults to false)
+# add a "Review in Gitpod" button to pull requests (defaults to false)
     addBadge: false
-    # add a label once the prebuild is ready to pull requests (defaults to false)
+# add a label once the prebuild is ready to pull requests (defaults to false)
```
Comment (thegedge) [inline .gitpod.yml]: @JesterOrNot These indentation changes seem unrelated/unnecessary? 🙂

Comment (JesterOrNot) [reply .gitpod.yml]: Autoformatting sorry 😅

Comment (JesterOrNot) [reply .gitpod.yml]: Fixed

**Comment by thegedge** [comment]
Thanks, @JesterOrNot 🎉
